docs/success_metrics.md

```diff
@@ -0,0 +1,23 @@
+## Success Metrics
+
+DPUL & DPUL Collections is a success if patrons are inspired towards transformative use of PUL Material, whether they knew we had that material or not.
+
+To measure that success, we have the following metric vectors:
+
+1. Downloads over time (Up is better)
+    * Download is a pre-requisite for many use patterns.
+1. Interactions per session (Up is better)
+    * Engaged browse of our material is more likely to lead to inspiration, use, further research, and knowledge creation.
+1. Sessions greater than 1 minute (Up is better)
+    * Engaged use of material is more likely to lead to using it.
+1. Number of sources linking to our materials (Up is better)
+    * Transformative use via writing and research often includes linking back to our resources, and generally sharing resources through various channels will increase the likelihood of folks using it.
+1. Bounce rate (Down is better)
+    * Bounce rate measures how often users land on our site and then leave without further interaction. We're hoping for users to be engaged with material on this site and find new resources - the less often they leave immediately, the more we expect that to happen.
+
+These measures can be tracked at https://figgy.princeton.edu/reports/dpul_success_dashboard.
+
+There are a couple of metrics that we'd like to measure but have not yet found a way to do. If we find a way to do these, we think we'd be better able to measure success:
```

Comment:
```suggestion
There are a couple of metrics that we'd like to observe but have not yet found a way to measure. If we find a way to do these, we think we'd be better able to measure success:
```

Reply:
alternative suggestion: There are a couple of metrics that we think would enable us to better measure success, but we haven't found a way to implement them yet.